### ISSUE TITLE
Center Loss 使用方法求助

### ISSUE THREAD

**Issue by caikw0602** [PaddlePaddle/Paddle#24917]
现有图像分类问题，想使用交叉熵+Center loss作为损失函数，试着用了一下paddle的centerloss，但发生了错误，请问如何使用呢，能否提供一点使用案例？
自己试用指令如下：
![1](https://user-images.githubusercontent.com/45365909/83825357-b4526900-a70b-11ea-90a4-bd6c62e31cef.png)
错误提示如下：
![2](https://user-images.githubusercontent.com/45365909/83825404-d2b86480-a70b-11ea-9028-ab2a79addb4a.png)
![3](https://user-images.githubusercontent.com/45365909/83825412-d51abe80-a70b-11ea-8762-05fa2d24a888.png)
![4](https://user-images.githubusercontent.com/45365909/83825415-dba93600-a70b-11ea-94b7-b8297818e440.png)
期待回复，谢谢

**Comment by LDOUBLEV** [comment]
> 现有图像分类问题，想使用交叉熵+Center loss作为损失函数，试着用了一下paddle的centerloss，但发生了错误，请问如何使用呢，能否提供一点使用案例？
> 自己试用指令如下：
> ![1](https://user-images.githubusercontent.com/45365909/83825357-b4526900-a70b-11ea-90a4-bd6c62e31cef.png)
> 错误提示如下：
> ![2](https://user-images.githubusercontent.com/45365909/83825404-d2b86480-a70b-11ea-9028-ab2a79addb4a.png)
> ![3](https://user-images.githubusercontent.com/45365909/83825412-d51abe80-a70b-11ea-8762-05fa2d24a888.png)
> ![4](https://user-images.githubusercontent.com/45365909/83825415-dba93600-a70b-11ea-94b7-b8297818e440.png)
> 期待回复，谢谢

我写了个单测，没有问题，你可以试一下，我的paddle版本是1.7
```
def test_centerloss():
    import paddle.fluid as fluid
    train_program = fluid.Program()
    start_program = fluid.Program()

    with fluid.program_guard(train_program, start_program):

        input = fluid.layers.data(name='x', shape=[2, 30], dtype='float32')
        label = fluid.layers.data(name='y', shape=[2, 1], dtype='int64')
        alpha = 0.01
        center_loss = fluid.layers.center_loss(input=input,
                                           label=label,
                                           num_classes=10,
                                           alpha=alpha,
                                           param_attr=fluid.initializer.Xavier(uniform=False),
                                           update_center=True)
        x = np.random.uniform(0, 1, [2, 30]).astype(np.float32)
        y = np.random.randint(0, 10, [2, 1]).astype(np.int64)

        use_gpu = False
        place = fluid.CUDAPlace(0) if use_gpu else fluid.CPUPlace()
        exe = fluid.Executor(place)
        exe.run(program=fluid.default_startup_program())

    compiled_prog = fluid.compiler.CompiledProgram(train_program)
    out = exe.run(compiled_prog,
                       feed={"x": x, "y": y},
                       fetch_list=[center_loss.name])
    print(out)

```
